src/utils.sh

```diff
@@ -301,25 +307,25 @@ function prepare_apt() {
             if [[ $exit_code != 0 ]];
             then
                 log_error "prepare_apt() step 2 failed with error %d" exit_code
-                rm -f /etc/apt/sources.list.d/microsoft-prod.list &> /dev/null
+                rm -f /etc/apt/sources.list.d/microsoft-prod.list 2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT
```

Comment:
>2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT [](http://example.com/codeflow?start=66&length=39)

nit: as we use this redirection a few times in our code, do we want to create a macro for it?